homeassistant/components/nuki/__init__.py

```diff
@@ -57,6 +50,16 @@
 UPDATE_INTERVAL = timedelta(seconds=30)
 
 
+@dataclass
```

Comment:
```suggestion
@dataclass(slots=True)
```

If you really want to reduce memory some more